cmake TARGET_LINK_DIRECTORIES in version 3.13

Begin to build miniob
-- This is PROJECT_BINARY_DIR dir /root/miniob/build/src
-- This is PROJECT_SOURCE_DIR dir /root/miniob/src
Begin to build obclient
CMake Error at src/obclient/CMakeLists.txt:11 (TARGET_LINK_DIRECTORIES):
  Unknown CMake command "TARGET_LINK_DIRECTORIES".


-- Configuring incomplete, errors occurred!
See also "/root/miniob/build/CMakeFiles/CMakeOutput.log".
[root@host-10-208-65-5 build]# cmake -version
cmake version 3.12.1

TARGET_LINK_DIRECTORIES can be use in version >=3.13

--- a/src/obclient/CMakeLists.txt
+++ b/src/obclient/CMakeLists.txt
@@ -8,8 +8,6 @@ TARGET_INCLUDE_DIRECTORIES(obclient PRIVATE . ${PROJECT_SOURCE_DIR}/../deps /usr
 # 父cmake 设置的include_directories 和link_directories并不传导到子cmake里面
 #INCLUDE_DIRECTORIES(BEFORE ${CMAKE_INSTALL_PREFIX}/include)
 
-TARGET_LINK_DIRECTORIES(obclient PRIVATE /usr/local/lib ${PROJECT_BINARY_DIR}/../../lib)
-
 # stdio.h 必须放在readline/readline.h 前面，因为readline头文件不能直接单独编译
 CHECK_INCLUDE_FILES("stdio.h;readline/readline.h" HAVE_READLINE_HEADER)
 

